airgun/views/repository.py

```diff
@@ -172,13 +201,17 @@ class YumRepository(View):
             name='Upstream Authorization')
         metadata_type = EditableEntrySelect(name='Yum Metadata Checksum')
         mirror_on_sync = EditableEntryCheckbox(name='Mirror on Sync')
-        ignore_global_proxy = EditableEntryCheckbox(
-            name='Ignore Global HTTP Proxy')
         publish_via_http = EditableEntryCheckbox(name='Publish via HTTP')
         gpg_key = EditableEntrySelect(name='GPG Key')
         download_policy = EditableEntrySelect(name='Download Policy')
         upload_content = FileInput(name='content[]')
         upload = Text("//button[contains(., 'Upload')]")
+        http_proxy_policy = EditableEntrySelect(name='HTTP Proxy')
+        proxy_policy = ConditionalSwitchableView(reference='http_proxy_policy')
+
+        @proxy_policy.register(True, default=True)
+        class NoSpecificHttpProxy(View):
+            pass
```

Comment:
Please mentioned the comment, why we are just doing pass here?

Reply:
You can find the same pattern used in `airgun/views/host.py`, `airgun/views/job_invocation.py` or `airgun/views/virtwho_configure.py`.

The thing is - `ConditionalSwitchableView` expects to find a subclass matching current value of reference element. If there is nothing matching, subclass registered with `default=True` will be used. So there must be one like that.

That subclass is empty, because new fields appear only for one value of reference field. Two other possible values do not result in any fields appearing.

I agree this pattern might be surprising, as `ConditionalSwitchableView` is complex and not used often. But since other usages of that pattern are not commented, I don't think we need comment here.

Reply:
@mirekdlugosz thanks for mentioning the reason.